scripts/gulpfiles/git_tasks.js

```diff
@@ -88,7 +93,7 @@ const recompile = gulp.series(
 // Create and push an RC branch.
 // Note that this pushes to google/blockly.
 const createRC = gulp.series(
-  syncDevelop,
+  syncDevelop(),
```

Comment:
?

This is supposed to be the function, not the call to it. gulp.series should call it.

Reply:
I was getting an error because done wasn't getting called. I think the syncDevelop function is returning the function that needs to be called.

Reply:
Hm, I guess it just wasn't working before and no one noticed? Does that mean this wasn't ever being used?

Reply:
I used it in the January release where it was working because it looked like this: `gulp.task('git-sync-develop', syncBranch('develop'));` And then I think it got refactored and I didn't use it in the last release because we changed how we were releasing.